sdk/core/Azure.Core/src/Shared/PropertyReferenceTypeAttribute.cs

```diff
@@ -14,23 +14,23 @@ internal class PropertyReferenceTypeAttribute : Attribute
         /// <summary>
         /// Instatiate a new reference type attribute.
         /// </summary>
-        /// <param name="skipTypes"> An array of types to skip for this reference type. </param>
-        public PropertyReferenceTypeAttribute(Type[] skipTypes)
+        /// <param name="optionalProperties"> An array of property names that are optional when comparing the type. </param>
+        public PropertyReferenceTypeAttribute(string[] optionalProperties)
```

Comment:
This is a breaking change, but nothing actually uses `skipTypes`.

Reply:
It seems this is an internal class change and not breaking the ApiCompatVersion check.